apps/vs-code-designer/src/app/commands/workflows/exportLogicApp.ts

```diff
@@ -83,6 +83,7 @@ class ExportEngine {
       const templateExists = await fse.pathExists(templatePath);
       if (!this.resourceGroupName || !templateExists) {
         this.setFinalStatus('Succeeded');
+        this.addStatus(localize('workflowsExportedSuccessfully', 'The selected workflows exported successfully.'));
```

Comment:
Can we add this - `localize('workflowsExportedSuccessfully', 'The selected workflows exported successfully.')` to a constant and use it all the places instead?

Reply:
Sure! thanks